Sources/ChangeLogGenerator/Generator/Generator.swift

```diff
@@ -190,10 +225,12 @@ public struct Generator {
                 entry = ChangelogEntry(tag: tag.name, pullRequests: [])
             }
 
-            let shouldFilterOut = expression.map {
+            var shouldFilterOut = expression.map {
                 pullRequest.simpleMessage.matches(regularExpression: $0)
```

Comment:
Just to confirm is `simpleMessage` the PR title?

Reply:
That's the formatted title that shows up in the change log https://github.com/tahirmt/changelog-generator/blob/8889ed52e55e57c8033bb15b06688a348176883d/Sources/ChangeLogGenerator/Models/PullRequest.swift#L28

Reply:
Perhaps I should just use the `title` here

Reply:
Ahh I see. Hmm what about `changelogEntry`?

Reply:
I meant for filtering I think it makes sense to use `title` instead of `simpleMessage`. And on your other point it does make sense to rename the `simpleMessage` to be more meaningful like `formattedMessage` or `changelogEntry` works just as well.